annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/method/MethodDeclarationTree.java

```diff
@@ -46,7 +48,9 @@ public class MethodDeclarationTree extends MetaData<MethodNode> {
   private HashMap<Integer, MethodNode> nodes;
 
   /** Set of all classes flat name declared in module. */
-  private HashSet<String> classNames;
+  private Multimap<String, MethodNode> classConstructorMap;
```

Comment:
As a side comment, why is this class called `MethodDeclarationTree`?  I don't see how a tree is involved.  Maybe in a follow-up we can rename to `MethodDeclarationStore`?

Reply:
We changed the name once through #54 but even the new name is not an appropriate name. The tree structure is within the nodes that each node's parent is the immediate super method. After release of `1.3.6` , I am planning to remove this data structure along others such `FIeldDeclarationStore` as they are internal data structures of Annotator and can be replaced with existing ones.